### ISSUE TITLE
Random Catch Time

### ISSUE THREAD

**Issue by Inuyushi** [svanheulen/fisher-windower-addon#12]
This add-on is amazing! Well done. I'm still testing it out and learning the details of how this operates, but so far is still works.

One suggestion, which I would love to know how to do, and would really improve the operation of this program: Can a random time duration be added to the kill packet? This would make it harder to detect on SE's side that someone is using an outgoing packet. As it stands, it's a constant 3 or 5 or 6 seconds. If a variable was added to randomize that time by +2~4 seconds, or longer, it would be amazing.

I know you don't play FFXI anymore, but this added feature would be amazing. I'll try and do it myself, but if you know how to do it real quick that would be so sweet.

**Comment by svanheulen** [comment]
It should be easy enough using the [math.random](http://www.lua.org/manual/5.1/manual.html#pdf-math.random) function. Just find where I send the "wait" commands and add a random amount to the delay. Of course it would also be good to include an option to enable/disable this feature.

I say fork my repo and give it a shot yourself. If you have any questions or run into a problem I'll be glad to help. Then when it's all done you can send me a pull request.

**Comment by Inuyushi** [comment]
Ok, it didn't take long cause your code is really clean. I forked the code and submitted a pull request so others can enjoy. I have not fully tested the code to make sure it does not break, however the add-on does load. I hit full fatigue already today while working on the code. I'll test it out tomorrow. If anything breaks, I'll re-submit a pull request. I'm just submitting the first to have it reviewed by you. This is my first time ever using Github for something like this, so sorry if I messed up anywhere!
